rosa_getting_started/rosa-sts-required-aws-service-quotas.adoc

```diff
@@ -0,0 +1,12 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="rosa-sts-required-aws-service-quotas"]
+= Required AWS service quotas
+:context: rosa-sts-required-aws-service-quotas
+toc::[]
+
+Review this list of the required Amazon Web Service (AWS) service quotas that are required to run an {product-title} cluster.
```

Comment:
@JQ WDYT >  "Review this list of the Amazon Web Service (AWS) service quotas that are required to run an {product-title} cluster."
I can make changes to both files if you approve of the change.

Reply:
I'm confused - we talked about needing to reuse the original assembly and not create a second version for STS. Then just using Xrefs in the STS section to point to the original assembly (instead of trying to include it twice in the YAML file).

Reply:
We can work on mechanics after dev review.